main.tf

```diff
@@ -55,28 +54,28 @@ resource "ibm_resource_key" "cos_credentials" {
   role                 = local.role
   parameters           = { "HMAC" = true }
 
-  //User can increase timeouts
+  # User can increase timeouts
   timeouts {
     create = "15m"
     delete = "15m"
   }
 }
 
 module "clis" {
-  source = "cloud-native-toolkit/clis/util"
-  version = "1.16.4"
+  source  = "cloud-native-toolkit/clis/util"
+  version = "1.18.1"
 
   clis = ["jq"]
 }
 
-data external instance {
+data "external" "instance" {
```

Comment:
@triceam @seansund - Is this script still needed. What was the context for its addition? I assume it's trying to work some limitation in the provider? We've not seen issue with waiting for the cos instance so far in PaaS, so wondering whether this is still valid.